app/components/Name/Name.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import "react-slideshow-image/dist/styles.css";
+import { Fade, Zoom } from "react-slideshow-image";
+
+export default function Name() {
+  const words = ["Hello, there!", "Привет!", "Ciao!", "Selam!", "こんにちは！"];
+  const names = ["Фаузи Нажжап", "فوزي نجار"];
+  const zoomInProperties = {
+    duration: 4000,
+    transitionDuration: 1000,
+    pauseOnHover: false,
+    arrows: false,
+    autoplay: true,
+    scale: 1,
+  };
+
+  return (
+    <>
+      <Fade {...zoomInProperties}>
```

Comment:
no need for spreading, same as above.
it's better to pass component props directly if they are fixed values.